src/schemas/localization.json

```diff
@@ -30,6 +33,10 @@
         "Locale": {
             "type": "string",
             "pattern": "^[a-zA-Z]+([a-zA-Z0-9\\-]*)$"
+        },
+        "TimeZone": {
+            "type": "string",
+            "pattern": "^[-+_\/ A-Za-z 0-9]*$"
```

Comment:
Shouldn't we make sure there is a slash in it?

Reply:
According to the [TZ database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), most but not all have a slash.